templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -156,6 +156,8 @@ export class {{ service.name }}Client {
     }
     if (!opts.fallback) {
       clientHeader.push(`grpc/${this._gaxGrpc.grpcVersion}`);
+    } else {
+      clientHeader.push(`rest/${this._gaxGrpc.grpcVersion}`);
```

Comment:
Shouldn't the value of this be something else, something that refers to the (language-specific) REST encoding version, instead of `grpcVersion`? Or does this single number encode both?

[In the latter case, let's at least make a note of that (ideally, changing the name to be more generic, but that might be a more far-reaching change that can handled separately, later, so a comment at least would be good)]

Reply:
This is gax-nodejs existing implementation. I think it is worth to think to refactor the grpcClient and isolate fallback or rest from grpc.
But it is not block this task. I will revisit the evaluation. Thanks @vchudnov-g

Reply:
Could you generate a comment clarifying that even though the variable is called `grpcVersion`, it actually captures some non-gRPC info X (where X could be "gax version implementing HTTP" or whatever is an accurate description)

(Someone perusing the code would assume based on the name we're sending the wrong info in the REST case, so it helps to have a comment saying that this is intentional and the variable name is misleading. And then we can fix the name or reorganize the code later.)